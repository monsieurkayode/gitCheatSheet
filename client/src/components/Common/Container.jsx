import styled from 'styled-components';

export default styled.main`
  max-width: 740px;
  margin-top: 5%;
  margin-bottom: 60px;
  padding: 0 1%;
  flex: 1;
  width: 100%;
  z-index: 100;

  @media only screen and (max-width: 424px) {
    align-self: normal;
    margin-top: 10%;
    padding: 0 2%;
  }
`;
