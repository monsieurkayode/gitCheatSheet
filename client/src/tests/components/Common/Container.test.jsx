import React from 'react';
import { mount } from 'enzyme';
import Container from '../../../components/Common/Container';

describe('Container Component', () => {
  const wrapper = mount(<Container />);

  it('should render correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
