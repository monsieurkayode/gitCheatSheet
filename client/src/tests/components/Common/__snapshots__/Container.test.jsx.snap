// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container Component should render correctly 1`] = `
<styled.main>
  <StyledComponent
    forwardedComponent={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "fosNZJ",
          "rules": Array [
            "
  max-width: 740px;
  margin-top: 5%;
  margin-bottom: 60px;
  padding: 0 1%;
  flex: 1;
  width: 100%;
  z-index: 100;

  @media only screen and (max-width: 424px) {
    align-self: normal;
    margin-top: 10%;
    padding: 0 2%;
  }
",
          ],
        },
        "displayName": "styled.main",
        "foldedComponentIds": Array [],
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "main",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <main
      className="sc-bdVaJa fosNZJ"
    />
  </StyledComponent>
</styled.main>
`;
